msl/equipment/resources/vaisala/ptb330.py

```diff
@@ -19,11 +19,19 @@ class PTB330(PTU300):
 
     def __init__(self, record: EquipmentRecord) -> None:
         """Vaisala Barometer PTB330 series.
-        Device manual is available `here <https://docs.vaisala.com/v/u/M210855EN-D/en-US>`_.
+        THe device manual is available `here <https://docs.vaisala.com/v/u/M210855EN-D/en-US>`_.
```

Comment:
A nitpicky comment &dash; replace "THe" with "The"

Reply:
Doh... last minute change and I got almost half the letters wrong!